docs/concepts/simulation.md

```diff
@@ -0,0 +1,72 @@
+# Cosmos Blockchain Simulator
```

Comment:
The simulation concept doc is going to be under the "Using the SDK" section, so maybe relocate in `docs/concepts/using-the-sdk/simulation.md`